mlrun/mlutils/models.py

```diff
@@ -39,7 +39,7 @@ def gen_sklearn_model(model_pkg, skparams):
     else:
         model_config = get_class_fit(model_pkg)
 
-    for k, v in skparams:
+    for k, v in skparams.items():
```

Comment:
I see several functions are using that function and giving dict items in the `skparams` which will fail if you'll do `.items()` on, for example [sklearn classifier](https://github.com/mlrun/functions/blob/master/sklearn_classifier/sklearn-classifier.py#L93)
So we must be backwards compatible here, here's what I suggest:
```suggestion
    # we used to use skparams as is (without .items()) so supporting both cases for backwards compatibility
    skparams = skparams.items() if isinstance(skparams, dict) else skparams
    for k, v in skparams:
```